docs/authoring.md

```diff
@@ -90,6 +90,7 @@ This section describes the steps needed to consume a C#/WinRT component for the
 - [C# desktop applications](#consuming-from-c-applications) (.NET 5 or later)
 - [Packaged applications](#Consuming-from-packaged-applications)
 - [Out of process components](#Consuming-an-out-of-process-component)
+- [Windows App SDK Support](#Using-Windows-App-SDK)

```

Comment:
How about "Windows App SDK applications"? 

We may also want to note that these are C# and C++ desktop applications/distinguish these from the first 2 scenarios.